data/clients.json

```diff
@@ -325,6 +325,16 @@
 			"category": "Microsoft Azure",
 			"comments": "(Compatible with any App Service)"
 		},
+		{
+			"name": "njs-acme",
+			"url": "https://github.com/nginx/njs-acme",
+			"acme_v2": "acme >= 2.0.0",
```

Comment:
This can just say "true". Other clients use it to indicate which versions of those clients support acmev2; it looks like all versions of njs-acme support acmev2.

Reply:
done. thank you!